# Exercícios com if

## 1.Cálculo de bônus
Crie um programa que calcule e dê um print no bônus que os funcionários devem receber segundo a regra:

A meta é 1000 vendas.

Se o valor de vendas for maior ou igual a meta, o valor do bônus do funcionário é 10%.          
Caso contrário o valor do bônus é 0         
Print o bônus dos 3 funcionários.       

In [1]:
vendas_funcionario1 = 1000
vendas_funcionario2 = 770
vendas_funcionario3 = 2700

if vendas_funcionario1 >= 1000:
    bonus = 0.1 * vendas_funcionario1
    print(f"O bônus foi de {bonus}.")
else:
    print("O funcionário não ganhou bônus")

if vendas_funcionario2 >= 1000:
    bonus = 0.1 * vendas_funcionario2
    print(f"O bônus foi de {bonus}.")
else:
    print("O funcionário não ganhou bônus")

if vendas_funcionario3 >= 1000:
    bonus = 0.1 * vendas_funcionario3
    print(f"O bônus foi de {bonus}.")
else:
    print("O funcionário não ganhou bônus")

O bônus foi de 100.0.
O funcionário não ganhou bônus
O bônus foi de 270.0.


Resposta:
O funcionário 1 ganhou 100 de bônus
O funcionário 2 ganhou 0 de bônus
O funcionário 3 ganhou 270 de bônus

### 2.Cálculo de bônus com uma nova regra
Agora, crie um novo código que calcule e dê um print no bônus dos funcionários novamente. Porém há uma nova regra nesse 2 caso:

A meta é 1000 vendas
Agora os funcionários que venderem muito acima da meta ganham mais bônus do que os outros. Então o bônus é definido da seguinte forma:

Se vendas funcionário for maior ou igual a 2000, então o bônus é de 15%.        
Se for menor do que 2000 e maior ou igual a 1000, então o bônus é 10%.    
Se for menos que 1000 não ganha bônus.

Use as mesmas variáveis do exercício anterior.

In [5]:
if vendas_funcionario1 >= 1000 and vendas_funcionario1 <= 2000:
    bonus = 0.1 * vendas_funcionario1
    print(f"O bônus foi de {bonus}.")
elif vendas_funcionario1 >= 2000:
    bonus = 0.15 * vendas_funcionario1
    print(f"O bônus foi de {bonus}.")
else:
    print("O funcionário não ganhou bônus")

if vendas_funcionario2 >= 1000 and vendas_funcionario2 <= 2000:
    bonus = 0.1 * vendas_funcionario2
    print(f"O bônus foi de {bonus}.")
elif vendas_funcionario2 >= 2000:
    bonus = 0.15 * vendas_funcionario2
    print(f"O bônus foi de {bonus}.")
else:
    print("O funcionário não ganhou bônus")

if vendas_funcionario3 >= 1000 and vendas_funcionario3 <= 2000:
    bonus = 0.1 * vendas_funcionario3
    print(f"O bônus foi de {bonus}.")
elif vendas_funcionario3 >= 2000:
    bonus = 0.15 * vendas_funcionario3
    print(f"O bônus foi de {bonus}.")
else:
    print("O funcionário não ganhou bônus")

O bônus foi de 100.0.
O funcionário não ganhou bônus
O bônus foi de 405.0.


Resposta:
O funcionário 1 ganha 100 bônus
O funcionário 2 ganhou 0 de bônus
O funcionário 3 ganhou 405 de bônus